compiler/cpp/src/thrift/generate/t_go_generator.cc

```diff
@@ -152,6 +152,10 @@ class t_go_generator : public t_generator {
                                  const string& tstruct_name,
                                  bool is_result = false,
                                  bool uses_countsetfields = false);
+  void generate_go_struct_deepequal(std::ostream& out,
```

Comment:
Can we rename all `*deepequal*` into `*equals*`? I don't think it's accurate to call them "deepequal".

Reply:
Thanks for review, *equals* sounds better.